packages/patternlab/styleguide/source/_patterns/03-organisms/feedback/mass-feedback.twig

```diff
@@ -13,78 +14,75 @@
 
     <div class="ma__mass-feedback-form__fields">
       <fieldset class="ma_feedback-fieldset">
```

Comment:
Let's remove the `fieldset` since it's not necessary in this case as we've replaced the `legend` with `label`.